DESCRIPTION

```diff
@@ -72,6 +72,7 @@ Suggests:
     R6,
     RJSONIO,
     rmarkdown,
+    rnaturalearth,
```

Comment:
Overall, looks good! I think that you could add this to config/needs/website instead and remove geojsonio from there (if not used elsewhere)